lib/atom-cfn-lint.js

```diff
@@ -108,8 +117,17 @@ module.exports = {
         'atom-cfn-lint.overrideSpecPath',
         (value) => { this.overrideSpecPath = value }
       ),
+      atom.config.observe(
+        'atom-cfn-lint.extraGrammarScopes',
+        (value) => { this.extraGrammarScopes = value }
```

Comment:
This will only work when the package is initially loaded. You will need to handle _changes_ in this array in the `observe` callback here.

Over in `linter-eslint` we take a slightly [different approach](https://github.com/AtomLinter/linter-eslint/blob/9a64dae996bd8cea686ef1ddb5c08bc44a1094a3/src/main.js#L110-L113) where we set the default value of the equivalent config option to be the supported list. This _does_ allow the user to remove supported ones, but as they are only adding them most of the time we haven't had any issues with it.

_Note: Removing the contents of the Array and adding the new values isn't the most efficient, but it's super simple and when there are only a few (<10) entries in the Array it isn't exactly a performance bottleneck._

Reply:
@default50 are you able to make these changes?  If not I can provide some help.  Thanks for pull request.